### ISSUE TITLE
[Twitter Lab 1.2]  Styling, Ordering, Create

### ISSUE THREAD

**Issue by ojgao** [bitprj/curriculum#603]
12.md/121.md/122.md
- these are less of steps associated with card 1 and more so steps that need to be done before hand. Create a new card with this information and name it 0.md

54.md/541.md
- show desired result on easy card
- move the completed code to medium card
